jenkins/requirements.txt

```diff
@@ -0,0 +1,8 @@
+setuptools==36.0.1
+pyinstaller==3.5
```

Comment:
`pyinstaller==3.5`
i think can be removed

Reply:
yes, will remove.

Reply:
done